app/api/dao/mentorship_relation.py

```diff
@@ -92,7 +92,17 @@ def create_mentorship_relation(self, user_id: int, data: Dict[str, str]):
         )
         for relation in all_mentor_relations:
             if relation.state == MentorshipRelationState.ACCEPTED:
+                if relation.mentee_id == mentee_id:
+                    return (
+                        messages.RELATION_REQUEST_ALREADY_ACCEPTED,
+                        HTTPStatus.CONFLICT,
+                    )
                 return messages.MENTOR_ALREADY_IN_A_RELATION, HTTPStatus.BAD_REQUEST
+            elif (
```

Comment:
Nit: you don't really need elif over here

Reply:
I didn't understand why isn't it required?

Reply:
You can just use another if statement right ?  No need for the elif

Reply:
I mean I can but why, if the relation.state is ACCEPTED it cannot be PENDING then , only one prevails at a time. Do you still want me to use if instead of elif?

Reply:
I agree with what @decon-harsh says here. @epicadk why we need an if here?